omnibus/cookbooks/omnibus-supermarket/test/integration/default/inspec/controls/install-check.rb

```diff
@@ -31,7 +31,23 @@
 
   describe file('/var/opt/supermarket/nginx/etc/sites-enabled/rails') do
     its(:content) { should match /ssl_dhparam/ }
- end
+  end
+end
+
+control "FIPS" do
+  title "FIPS 140-2 enablement"
+
+  only_if { property['supermarket']['fips_enabled'] }
+
+  %w(nginx rails sidekiq).each do |service_name|
+    describe file("/opt/supermarket/service/#{service_name}/run") do
+      its(:content) { should match(/export OPENSSL_FIPS=1/) }
+    end
+  end
+
+  describe file('/var/opt/supermarket/nginx/etc/sites-enabled/rails') do
+    its(:content) { should match(/ssl_ciphers FIPS/) }
+  end
 end
```

Comment:
Can we add validations for the available ciphers as well?

Reply:
Probably! Like, check the whole line `ssl_ciphers FIPS@STRENGTH:!aNULL:!eNULL` and not just cut it off at `FIPS`?

Reply:
@marcparadise Re-reading this. Or do you mean adding a check for what the ciphers should be in the non-FIPS case?

Reply:
Checking for the nginx list of ciphers (including FIPs) is correct in the FIPs case; I think we're only checking the env var for nginx here.